Formula/avrdude.rb

```diff
@@ -30,46 +31,37 @@ class Avrdude < Formula
   end
 
   head do
-    url "https://svn.savannah.nongnu.org/svn/avrdude/trunk/avrdude"
+    url "https://github.com/avrdudes/avrdude.git", branch: "main"
 
-    depends_on "autoconf" => :build
-    depends_on "automake" => :build
-    depends_on "libtool" => :build
+    depends_on "cmake" => :build
   end
 
-  depends_on "automake" => :build
   depends_on "hidapi"
-  depends_on "libftdi0"
-  depends_on "libhid"
-  depends_on "libusb-compat"
+  depends_on "libftdi"
+  depends_on "libusb"
 
-  uses_from_macos "bison"
-  uses_from_macos "flex"
+  uses_from_macos "bison" => :build
+  uses_from_macos "flex" => :build
 
   on_macos do
-    depends_on "libelf"
+    depends_on "libelf" => :build
   end
 
   on_linux do
     depends_on "elfutils"
+    depends_on "readline"
   end
 
   def install
-    # Workaround for ancient config files not recognizing aarch64 macos.
-    am = Formula["automake"]
-    am_share = am.opt_share/"automake-#{am.version.major_minor}"
-    %w[config.guess config.sub].each do |fn|
-      chmod "u+w", fn
-      cp am_share/fn, fn
-    end
-
     if build.head?
-      inreplace "bootstrap", /libtoolize/, "glibtoolize"
-      system "./bootstrap"
+      system "cmake", "-S", ".", "-B", "build", *std_cmake_args
```

Comment:
Note that this only builds static lib. Was trying `-DBUILD_SHARED_LIBS=ON` but hit some undefined symbols and haven't had time to debug it. Though, only using `cmake` on HEAD since this is what upstream recommends while tarball only has autoconf files.

Reply:
Try `-undefined dynamic_lookup` for undefined symbols.